src/app/components/client/ExposureCard.tsx

```diff
@@ -133,42 +137,43 @@ export const ExposureCard = (props: Props) => {
           }`}
         >
           <p>
-            This site is selling and publishing{" "}
+            This site is selling and publishing {" "}
             <a href={exposureDetailsLink}>
-              details about you.{" "}
+              details about you.
               <span>
-                <OpenInNew alt="" width="15" height="14" />
+                <OpenInNew alt="" width="13" height="13" />
               </span>
-            </a>{" "}
+            </a>
+            {" "}
             Remove this profile to protect your privacy.
           </p>
           <div className={styles.exposureListOfExposureTypes}>
             <ul>
               <li>Your exposed info:</li>
```

Comment:
This `<li>` feels like it's more a heading for the list, than that it is the first element of the list?

And I think for the rest, instead of a `<ul>`, you'll want a `<dl>`, and then the `<li>`s can be replaced by `<div>`s containing just the `<dt>`s and `<dd>`s.